dev/envvars.sh

```diff
@@ -1,16 +1,42 @@
 #!/bin/bash
 
-case "$(uname -s | tr '[:upper:]' '[:lower:]')" in
-  linux*)
-    if [ "$(ldd /bin/ls | grep -m1 'musl')" ]; then
-      DOTNET_RUNTIME_ID="linux-musl"
-    else
-      DOTNET_RUNTIME_ID="linux-x64"
-    fi
+# guess OS_TYPE if not provided
```

Comment:
This code is a copy from `./instrument.sh` we should look into unifying these scripts.

Reply:
Opened #2130